pages/_app.js

```diff
@@ -1,31 +1,23 @@
 import GlobalStyle from "../styles";
-import artPiecesData from "@/db/data_links";
-import { useEffect } from "react";
 import useLocalStorageState from "use-local-storage-state";
+import artPiecesData from "@/db/data";
+import { useEffect } from "react";
 
 export default function App({ Component, pageProps }) {
   const [artPieces, setArtPieces] = useLocalStorageState("artPieces", {
     defaultValue: [],
```

Comment:
```suggestion
    defaultValue: artPiecesData,
```
I think you can just use your initial art pieces here.

Reply:
I tried this one out already. When I clear the local storage and restart the localhost it does not set the data initially. But with the usEffect-hook it does.